data-jdbc/src/test/resources/application.yml

```diff
@@ -8,6 +8,7 @@ test-resources:
       startup-timeout: 300s
     mysql:
       startup-timeout: 300s
+      image-name: mysql:8.4.5
```

Comment:
Recently mysql docker started failing on CI (works locally on Mac and Rancher) as it seems it is using MySQL 9 so needed to hard-code back to an older 8.x version and it is working now.